### DISCUSSION TITLE
How does the GitHub Organizations work?

### DISCUSSION THREAD

**Discussion by cunhabruno** [artilleryio/artillery#3330] [Artillery]
Hello,

I have added a GitHub Organization in my Artillery Cloud org, but the public users in the GitHub org could not access the Artillery Cloud org, I might be missing something, would anyone know how this works or is there any doc about it?
<img width="963" alt="image" src="https://github.com/user-attachments/assets/e501b694-7c82-4e70-8dc1-f301aa19469b">
Many thanks!

**Comment by hassy** [comment]
hi @cunhabruno 👋 we'll take a look. Are the other (public) members of the org joining Artillery Cloud only able to see their own personal org, but not the one you created? (in the screenshot above)

**Reply by cunhabruno** [reply]
Yes correct, the other members can see only their personal org.

Thanks for the quick reply!

**Reply by hassy** [reply]
@cunhabruno we found a bug on our side, deploying a fix soon. In the meantime you should be able to add your colleagues directly by their GitHub username if you need to. Thanks for reporting the issue!
